cfme/tests/services/test_generic_service_catalogs.py

```diff
@@ -82,7 +82,15 @@ def test_service_generic_catalog_bundle(catalog_item):
     cells = {'Description': row_description}
     row, __ = wait_for(requests.wait_for_request, [cells, True],
         fail_func=requests.reload, num_sec=900, delay=20)
-    assert row.last_message.text == 'Request complete'
+    # Success message differs between 5.6 and 5.7
+    success_message = 'Service '
```

Comment:
I think this should be set inside the `if` statement since it isn't used for the <5.7 assert